android/app/src/primary/AndroidManifest.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:tools="http://schemas.android.com/tools">
-        
+
+    <!-- Required permission to use in-app billing. -->
+    <uses-permission android:name="com.android.vending.BILLING" />
```

Comment:
`tools:node="merge"` isn't required here?

Reply:
I'm not sure, but this is not necessary...
https://developer.android.com/studio/build/manifest-merge#node_markers